### PR TITLE
[JENKINS-68934] Mild page loading animation cleanup

### PR DESCRIPTION
**Pull request by janfaracik**
See [JENKINS-68934](https://issues.jenkins.io/browse/JENKINS-68934)

There was a duplicate class (`.behavior-loading`) that somehow crept in which was causing the loading overlay to be offset at the top. I've removed that class and done some mild cleanup (e.g. fixing the casing of some text).

https://user-images.githubusercontent.com/43062514/185237141-bfe26ba0-edd7-4009-b431-512965fde911.mov

The animation now (as initially intended in #6687) covers the navigation bar at the top.

### Proposed changelog entries

* N/A

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@daniel-beck @jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7012"></a>

### CODE REVIEW

File: core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy
Comment (timja) [inline core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy:24]: why update this place to `Loading` whereas in the other places add translations for `LOADING`?

**Review by timja — COMMENTED** [review]
Thanks! Tested locally and looks good

File: core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy
Comment (janfaracik) [reply core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy:24]: There was only the one instance of this (`core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index_it.properties`) so I changed the variable name too - happy to undo it though and add a separate default translation file.

Comment (timja) [reply core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy:24]: possibly something to follow up with and make them all consistent? Ideally `Loading`

**Review by timja — APPROVED** [review]

File: core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy
Comment (daniel-beck) [reply core/src/main/resources/jenkins/model/GlobalCloudConfiguration/index.groovy:24]: Could easily be a reusable control.

**Review by NotMyFault — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
